CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## 3.3.6 (2023-02-01)
+- Rebuilding with latest amazonlinux base image to address CVE issues.
```

Comment:
No need to include this since it's not a change to the daemon itself, this is implicit in the release process.